packages/headless/src/features/facets/category-facet-set/category-facet-set-slice.test.ts

```diff
@@ -121,6 +122,17 @@ describe('category facet slice', () => {
     expect(FacetReducers.handleFacetDeselectAll).toHaveBeenCalledTimes(1);
   });
 
+  it('dispatching #clearAllFacetValues calls #handleFacetDeselectAll for every category facet', () => {
+    jest.spyOn(FacetReducers, 'handleFacetDeselectAll');
+
+    state['1'] = buildMockCategoryFacetRequest();
+    state['2'] = buildMockCategoryFacetRequest();
+    state['3'] = buildMockCategoryFacetRequest();
+    categoryFacetSetReducer(state, clearAllFacetValues());
+
+    expect(FacetReducers.handleFacetDeselectAll).toHaveBeenCalledTimes(4);
```

Comment:
Why is `handleFacetDeselectAll` called 4 times? Shouldn't it be just 3?

Reply:
That's what I thought at first, but `spyOn` calls the function by default. 

https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname